cmd/main.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
I think we had the app in the `cmd/server/` subdirectory because we were intending on creating a TUI go client as well.

Reply:
My thinking was that the `tui` being a root level subdirectory as it's its own domain. Having it inside the `cmd` package feels like it's not of equal weight.

If we kept everything inside the `cmd` package would `cmd/tui` and then the root of that subdirectory is just the server?

If I'm not making sense I would like to discuss this a bit more